linkml_runtime/utils/schemaview.py

```diff
@@ -104,6 +105,19 @@ def is_absolute_path(path: str) -> bool:
     drive, tail = os.path.splitdrive(norm_path)
     return bool(drive and tail)
 
+def resolve_import(sn: str, i: str) -> str:
```

Comment:
if we're splitting this out to a separate function, it would be great to not have the arguments be named `sn` and `i`

Reply:
Have renamed to `source_sch` and `imported_sch`